ui/js/controllers/bugfiler.js

```diff
@@ -166,6 +168,11 @@ treeherder.controller('BugFilerCtrl', [
                 descriptionStrings += $scope.modalComment;
             }
 
+            var keywords = "";
+            if($scope.isIntermittent) {
+                keywords += "intermittent-failure";
```

Comment:
Various bugs make setting multiple keywords more difficult than it should be, so I'm not sure we should care too much about this right now:
There's no documentation (bug 1293780) about how to set keywords on initial bug creation via REST api, so I don't know exactly how to do it.
Setting `keywords` to a single keyword's name is what I'm currently doing and it apparently works for us.
Setting `keywords` to an array of keyword IDs will also work, but there's no API-exposed way (bug 1283782) to find those IDs, so it'd be a lot of trial and error to find the right ones.
From what I've been told, setting `keywords` to either an array or comma-separated list of keyword names will NOT work.